src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,198 @@ description: Notifications are messages that communicate information to the user
+
+## Toast notifications
+
+### Formatting
 
 <Row>
 <Column colLg={8}>
 
-![Toast notification example](images/notification-usage-2.png)
+![Example of toast notification with action button and time stamp](images/notification-toast-action.png)
 
 </Column>
 </Row>
 
-#### Inline notifications
+#### Action
+
+Toast notifications can include a link at the end of their body content. This link should be short and navigate users to a page or modal where they can take action to address the notification or find further information. Because toast notifications automatically dismiss, there should be alternative routes to navigate to the link destination.
+
+If you include links within the notification body that redirect the user to steps, be sure keyboard and screenreader users have enough time to access the notification or can easily find the notification after it disappears.
+
+#### Time stamp
+
+Toast notifications include a time stamp at the bottom the container. The time stamp shows the time the notification was sent. Using time stamps is optional but toasts should be consistent across the product so either all toasts should include time stamps or none of them should. The time stamp is optional and can be removed if a third line of content is needed.
+
+#### Sizing
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+Toast notifications use a fixed width and their height depends on the length of the notification message. As noted in the content guidelines, limit toast notifications to two lines of text.
+
+#### Placement
+
+Toast notifications slide in and out from the top right of the screen. They align to the edge of the last column and can stack with `$spacing-03` in-between. New toast notifications should appear at the top of the list, with older notifications being pushed down until they are dismissed.
+
+Actionable toast notifications do not appear on mobile screen widths.
 
 <Row>
 <Column colLg={8}>
 
-![Form example with inline notification](images/notification-usage-3.png)
+![Example of toast notification placement](images/notification-placement-toast.png)
 
 </Column>
 </Row>
+
+### Behavior
+
+#### Dismissal
+
+Toasts dismiss automatically after five seconds on the screen. They can also include a close button so users can dismiss them sooner. Toasts cover content on the screen so they should always be easily dismissed. Because toast notifications dismiss automatically, users should be able to access them elsewhere after the toast disappears. This allows them to be accessible for users who need more time reading the notification or who may want to refer back to the notification.
```

Comment:
> Because toast notifications dismiss automatically, users should be able to access them elsewhere after the toast disappears

Should we word this more strongly to say that there must be an alternate location if the toast is automatically dismissed?

Reply:
It definitely seems like it should be a requirement rather than a preference, right? (so "users _must_ be able to access..." rather than "users _should_...")

Reply:
Would simply changing "should be" to "must be" be enough?

Reply:
Yes, I think so.